src/cascadia/TerminalApp/userDefaults.json

```diff
@@ -6,6 +6,12 @@
 

```

Comment:
Add at very top

```
// This file was stamped by Terminal version 0.10.967
// It should still be usable in newer versions, but newer versions might have additional
// settings, help text, or changes that you will not see unless you clear this file
// and let us generate a new one for you.
// You can also always find all currently valid settings at <link>
```

Reply:
Hmm that requires some code ;P

Reply:
The version number was the whole point... I know it's code. I was hoping it wasn't that hard.